packages/bull-vault/test/e2e/FlashBullTestFork.t.sol

```diff
@@ -376,8 +327,8 @@ contract FlashBullTestFork is Test {
         uint256 ethInLendingBefore = IEulerEToken(eToken).balanceOfUnderlying(address(bullStrategy));
         uint256 usdcBorrowedBefore = IEulerDToken(dToken).balanceOf(address(bullStrategy));
         uint256 crabBalanceBefore = crabV2.balanceOf(address(bullStrategy));
-        uint256 ethToWithdrawFromBull = ethToWithdrawFromCrab.sub(maxEthForSqueeth).add(wethToWithdraw.sub(maxEthForUsdc));
-        userEthBalanceBeforeTx = user1.balance;
+        // uint256 ethToWithdrawFromBull = ethToWithdrawFromCrab.sub(maxEthForSqueeth).add(wethToWithdraw.sub(maxEthForUsdc));
```

Comment:
Should we delete these?

Reply:
let's keep it and try to find it how to do it

Reply:
Made comment in prior pr in how to calculate it - use uniswap quoter simulate a swap and see what you get